app/scripts/views/sub_panels.js

```diff
@@ -76,6 +77,10 @@ define(function (require, exports, module) {
 
       var view = self._createView(ChildView, {
         el: self.$(selector),
+        // Each child view receives its own model on creation. When
+        // the child view is shown in `showChildView`, it's model
```

Comment:
This comment is incorrect, the child view's model is updated by app.js before showChildView is called.